.azure-ci.yml

```diff
@@ -0,0 +1,200 @@
+trigger:
+- release/2.1*
+- release/2.2*
```

Comment:
Should this be batched? Usually the prs are merged all at once in release branches when the branch is open.

Reply:
Sure, that's probably fine.